### DISCUSSION TITLE
Change a field for multiple QSOs / Bulk edit

### DISCUSSION THREAD

**Discussion by raufaser** [magicbug/Cloudlog#2936] [Q&A]
Hi,

I want to edit a field for multiple QSOs? In my example I want to change the DOK for all stations of the recent 100FK activity from germany. They all have the DOK 100FK ... Is there anything like a bulk change form in cloudlog, or do I have to do it directly in the DB?

Thanks.

**Comment by magicbug** [comment]
You would need todo that using a SQL query in the database
